app/admin/reports/realtime/termination_distributions.rb

```diff
@@ -52,12 +52,13 @@ def find_collection
   end
 
   index as: :data, skip_drop_down_pagination: true do
-    data = collection.map do |x|
-      {
-        label: x.vendor.try!(:display_name) || "Vendor | #{x.vendor_id}",
-        value: x.originated_calls_count
-      }
-    end
+    data =
+      collection.map do |x|
+        {
+          label: x.vendor&.display_name || "Vendor | #{x.vendor_id}",
+          value: x.originated_calls_count
+        }
+      end
```

Comment:
disable cop that suggest such changes & undo this changes over all places in the project